tests/component_handlers/helm_wrapper/test_helm_wrapper.py

```diff
@@ -320,9 +320,10 @@ def test_raise_parse_error_when_helm_content_is_invalid(self):
         stdout = """---
             # Resource: chart/templates/test1.yaml
             """
-        with pytest.raises(ParseError) as value_error:
-            list(Helm.load_manifest(stdout))
-        assert str(value_error.value) == f"Failed to parse Helm stdout:\n {stdout}"
+        with pytest.raises(ParseError) as parse_error:
+            helm_template = list(Helm.load_manifest(stdout))
+            assert len(helm_template) == 0
+            assert str(parse_error.value) == "Not a valid Helm template source"
```

Comment:
assertion needs to be outside the context manager scope. this is a common mistake we used to make throughout the tests. see https://docs.pytest.org/en/7.1.x/how-to/assert.html#assertions-about-expected-exceptions

Reply:
or you could rewrite it as
```python
with raises(ParseError, match="Not a valid Helm template source"):
    ...
```